apps.toml

```diff
@@ -819,7 +820,7 @@ url = "https://github.com/YunoHost-Apps/django-fmd_ynh"
 [django-for-runners]
 added_date = 1674232499 # 2023/01/20
 category = "small_utilities"
-level = 0
+level = 1
```

Comment:
Hm. It's 8, see: https://ci-apps.yunohost.org/ci/apps/django-for-runners/latestjob

Reply:
```suggestion
level = 8
```